scripts/experiments/tanium_ben.py

```diff
@@ -51,7 +51,7 @@ def read_file(in_file: str):
     file_path = Path(in_file)
     data = []
     count = 0
-    with open(file_path) as fp:
+    with open(file_path, encoding='utf8') as fp:
```

Comment:
It doesn't really matter but all these utf8's could be const.FILE_ENCODING - but maybe you want to keep scripts separate from trestle modules.

Reply:
Fixed where the code is importing trestle. If not I have used utf-8 as we might end up with circular dependencies.